samples/display/lvgl/boards/native_posix.conf

```diff
@@ -2,3 +2,8 @@ CONFIG_SDL_DISPLAY=y
 CONFIG_SDL_DISPLAY_DEV_NAME="DISPLAY"
 CONFIG_SDL_DISPLAY_X_RES=320
 CONFIG_SDL_DISPLAY_Y_RES=240
+
+CONFIG_KSCAN=y
```

Comment:
Please move it to `boards/posix/native_posix/Kconfig.defconfig`, it already has DISPLAY configuration there. The same for `native_posix_64.conf`

Reply:
@jfischer-phytec-iot You mean CONFIG_KSCAN_SDL if CONFIG_KSCAN?

Reply:
@aescolar Yes similar as it was done for mimxrt boards https://github.com/zephyrproject-rtos/zephyr/pull/22055/commits/7ccea0841a3cdb9ede89daa776916b9f7473c9f8

Reply:
@jfischer-phytec-iot Does it make sense to add
```
config DISPlAY
	default y if LVGL

config KSCAN
	default y if LVGL
```
as well?

Reply:
I still do not agree to add new options to samples/*...*/boards/*.conf files. This way It has to be done for every sample that would use display and an input device. And on this architecture is will be always LVGL and CONFIG_LVGL_POINTER_KSCAN, there is nothing else that could be used with it.

Reply:
@jfischer-phytec-iot 

> I still do not agree to add new options to samples/.../boards/*.conf files

Actually the real problem here is that the basic LVGL sample has been reused to act as touchscreen sample, the best solution in this case is to split both samples and put `KSCAN` in the `prj.conf`.

> And on this architecture is will be always LVGL and CONFIG_LVGL_POINTER_KSCAN, there is nothing else that could be used with it.

I think `native_posix` should enable as least as possible features, e.g. if I would like to test LVGL without touchscreen support I don't want to be forced to disable KSCAN in my project config files.

Reply:
This sample is to simple to have two version, I suggest to move `CONFIG_KSCAN=y` to prj_kscan.conf